tests/osaka/eip7692_eof_v1/eip7873_tx_create/test_creation_tx.py

```diff
@@ -76,7 +74,6 @@ def test_legacy_create_tx_eof_initcode(
         to=None,
         gas_limit=100_000,
         data=smallest_initcode_subcontainer,
-        error=TransactionException.EOF_CREATION_TRANSACTION,
```

Comment:
The tx being invalid is in the [EIP-7873](https://eips.ethereum.org/EIPS/eip-7873#transaction-validation). I think you were using an incompatible evmone to fill https://github.com/ethereum/execution-spec-tests/pull/1451#issuecomment-2810125281 

CC @marioevz .

I think the solution may be to merge the evmone PR and then make a bugfix release of the tests with this resolved?

Reply:
We should not do this.  We are now changing a rule that will impact transaction pools.  We are taking a transaction that was once valid and are now making it invalid.  This will end poorly.

Reply:
This should then be discussed elsewhere (EIP7873 EthMag?). The test only reflects what's in the spec :shrug: